firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Util/Array.txt

```diff
@@ -0,0 +1,25 @@
+SCENARIO("Create an array", "[array]")
```

Comment:
Since this file tests the `make_array` function, please use structure the SCENARIOs/GIVENs/WHENs/THENs with the standard pattern we are using for functions ([documented on Notion](https://www.notion.so/pufferfish/Testing-13ce58d72a154b6883fbd57b71d37f76#51087c7b9af449d79163521b431a2e28)).